js/models/languagesMd.js

```diff
@@ -5280,6 +5280,418 @@ module.exports = Backbone.Model.extend({
+        Moderators: "Moderatorzy",
+        CurrentModerators: "Obecni moderatorzy",
+        AddModerators: "Dodaj nowych moderatorów",
+        DeselectToRemoveModerator: "Odznacz moderatorów, których chcesz usunąć",
+        SelectToAddModerator: "Zaznacz moderatorów, których chcesz dodać",
+        Categories: "Kategorie",
+        UpTo3: "Maksymalnie 3",
+        AboutYourStore: "Opis twojego sklepu",
+        PaymentType: "Metoda płatności",
+        ShipTo: "Wysyłka do",
+        FreeShipping: "Darmowa wysyłka",
+        OrderDetails: "Szczegóły zamówienia",
+        OrderSummary: "Podsumowanie zamówienia",
+        AllListings: "Wszystkie artykuły",
+        ComingSoon: "Wkrótce",
+        PaymentPending: "Płatność w toku",
+        FinalizePurchase: "Sfinalizuj transakcję",
+        LoadingImage: "Ładuję zdjęcie...",
+        UploadAvatar: "Wybierz avatar",
+        SaveAvatar: "Zapisz avatar",
+        NewAvatar: "Wybierz nowy avatar",
+        NewCoverImage: "Wybierz nowe tło",
+        Loading: "Ładowanie...",
+        Purchases: "Kupione",
+        Sales: "Sprzedane",
+        Cases: "Sprawy",
+        Enter: "Wprowadź",
+        Discover: "Odkryj",
+        Block: "Block",//
+        Unblock: "Unblock",//
+        Blocked: "Zablokowani",
+        Advanced: "Zaawansowane",
+        General: "Ogólne",
+        AllItems: "Wszystkie przedmioty",
+        DomesticShippingPrice: "Koszt wysyłki krajowej",
+        InternationalShippingPrice: "Koszt wysyłki międzynarodowej",
+        MinimumIs: "Minimum to",
+        Visibility: "Widoczność",
+        Title: "Tytuł",
+        DigitalItem: "Przedmiot cyfrowy",
+        PhysicalItem: "Przedmiot fizyczny",
+        DomesticShippingTime: "Termin wysyłki krajowej",
+        InternationalShippingTime: "Termin wysyłki międzynarodowej",
+        DisplayNSFWcontent: "Pokaż treści NSFW?",
+        Basic: "Basic",//
+        Content: "Zawartość",
+        StandardThemes: "Standardowe motywy",
+        NoPhotosAdded: "Brak zdjęć",
+        Summary: "Podsumowanie",
+        Funds: "Środki",
+        Discussion: "Dyskusja",
+        Quantity: "Ilość",
+        ShippingTo: "Wysyłka do",
+        ModeratedBy: "Nadzorowany przez",
+        Submit: "Wyślij",
+        maxLength20: "maksymalnie 20 znaków",
+        maxLength80: "maksymalnie 80 znaków",
+        maxLength200: "maksymalnie 200 znaków",
+        StoreModeratorsOptional: "Moderatorzy sklepu (opcjonalne)",
+        Searchformoderators: "Szukaj moderatorów",
+        Contributors: "Kontrybutorzy",
+        Support: "Pomoc",
+        Licensing: "Licencja",
+        Forward: "Dalej",
+        On: "Włączony",
+        Off: "Wyłączony",
+        ClickToChange: "Kliknij, by zmienić",
+        NotProvided: "niezapewniane",
+        NotFollowingAnyone: "Brak obserwowanych",
+        NoFollowers: "Brak obserwujących",
+        Moderator: "Moderator",//
+        moderatorSettings: {
+          ProvideResolution: "Provide dispute resolution",//
```

Comment:
For any phrase that isn't translated yet, we've been putting // not translated behind it, so it's easy to find and translate later.